gtclang/bundle/CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ yoda_find_package(
   REQUIRED_VARS dawn_DIR
   ADDITIONAL
     DOWNLOAD_DIR ${YODA_DOWNLOAD_DIR}
-    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../dawn
+    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../dawn/bundle

```

Comment:
this is the fix together with the change in the yoda branch

Reply:
Looks correct to me!